Client/Frontend/Settings/SearchBar/SearchBarSettingsViewModel.swift

```diff
@@ -104,9 +104,10 @@ private extension SearchBarSettingsViewModel {
         recordPreferenceChange(searchBarPosition)
 
         let notificationObject = [PrefsKeys.FeatureFlags.SearchBarPosition: searchBarPosition]
-        notificationCenter.post(name: .SearchBarPositionDidChange, object: notificationObject)
+        notificationCenter.post(name: .SearchBarPositionDidChange, anObject: nil)
```

Comment:
You should pass the object as a parameter here, since it was passed before
```suggestion
        notificationCenter.post(name: .SearchBarPositionDidChange, anObject: notificationObject)
```